docs/user/config/index.md

```diff
@@ -6,16 +6,28 @@ The later is preferred since it's more customizable, but it depends on your need
 
 ## Jest preset
 
+### The 3 presets
+
+`ts-jest` comes with 3 presets, allowing to cover most of project's base configuration:
+
+| Preset name | Description |
+|---|---|
+| `ts-jest/presets/default`<br>or `ts-jest` | `ts-jest` will take care of `.ts` and `.tsx` files only, leaving JavaScript files as-is. |
+| `ts-jest/presets/js-with-ts` | TypeScript and JavaScript file (`.ts`, `.tsx`, `.js` and `.jsx`) will be handled by `ts-jest` (and so TypeScript).<br>You'll need to set `allowJs` to `true` in your `tsconfig.json` file. |
```

Comment:
~(and so TypeScript)~
Other processors can be used now, right? So just saying that ts-jest processes it should be sufficient